jerry-core/CMakeLists.txt

```diff
@@ -641,9 +644,6 @@ jerry_add_define01(JERRY_SYSTEM_ALLOCATOR)
 
 # Valgrind
 jerry_add_define01(JERRY_VALGRIND)
```

Comment:
This code moved up. What about this define?

Reply:
The define is grouped in a list, there is no need move it.

The move of INCLUDE_THIRD_PARTY_VALGRIND is be cause we 
```
set(INCLUDE_CORE_PRIVATE ${INCLUDE_CORE_PRIVATE} PARENT_SCOPE) # for tests/unit-core
```
at the before

for define there is no such issue.
The move of define may trigger un-expected issue